int16 scale
int16 fram_rate
bool color_set